migrations/violet_background.js

```diff
@@ -0,0 +1,35 @@
+var migrationName = 'violet_backgrounds.js';
+
+/*
+ * This migration will find users with unlimited subscriptions who are also eligible for Jackalope mounts, and award them
+ */
+import Bluebird from 'bluebird';
+
+import { model as User } from '../website/server/models/user';
+
+async function addVioletBGs () {
+  let userCount = 0;
+  let promises = [];
+  let cursor = User.find({
+    'purchased.background.violet': {$ne: true},
```

Comment:
I agree with alys, it seems the query is not necessary, you could simply update all users with 

```
User.update({}, {$set: ...}).exec()